cmd/entrypoint-wrapper/main.go

```diff
@@ -326,9 +330,34 @@ func manageHome(proc *exec.Cmd) {
 		}
 	}
 	if needHome {
+		home = "/alabama"
 		// the last of any duplicate keys is used for env
-		proc.Env = append(proc.Env, "HOME=/alabama")
+		proc.Env = append(proc.Env, "HOME="+home)
+	}
+	return home
+}
+
+// manageGitConfig creates a default Git configuration file if necessary
+// This configuration will contain a `safe.directory` entry disabling the file
+// ownership verification (see `git-config(1)`).  If an existing configuration
+// file is found, no changes are made.
+func manageGitConfig(home string) error {
+	path := filepath.Join(home, ".gitconfig")
```

Comment:
In this solution it does not matter, but do you think in entrypoint it could be just always `/etc/gitconfig`?

Reply:
Not always, there is mysterious prefix before `/etc/gitconfig`